### DISCUSSION TITLE
Train model to detect keypoints at the EDGE of an object

### DISCUSSION THREAD

**Discussion by dalspark** [open-mmlab/mmpose#1809] [Ideas]
Hello,

I have successfully trained an HRNet model with an AP of 0.4999 using **hrnet_w48_coco_384x288_dark.py**. My dataset consists of 1000 training plus 250 validation images. The purpose of this model is to detect an area of interest at the furthest _edge_ of an object. 

For example, let's say I wanted to detect the coned point edge of a football. My keypoints would be annotated in a way were one pixel to the left is contained within the masking of the object, and one pixel to the right contains the background of the image.

Has anyone else approached a problem like this for estimating keypoints directly? The longer way would be to estimate a keypoint at the center + model a segmentation mask.

Thanks!

![image](https://user-images.githubusercontent.com/32441220/201546350-adc48028-2180-47b0-a7c0-ff9363b56f82.png)

**Comment by ly015** [comment]
Hi, this is really an interesting question. So far we don't have a model for tasks like this. We would like to convert this issue to a discussion so the community numbers can post their opinions.
